inttests/deploy_test.go

```diff
@@ -34,7 +34,7 @@ func TestDeployValidateMDMDetails(t *testing.T) {
 		"mdmUser":     "admin",
 		"mdmPassword": "Password123",
 	}
-	mapData["mdmIps"] = []string{"10.247.101.68"}
+	mapData["mdmIps"] = []string{"10.247.66.67"}
```

Comment:
we should not expose our internal ips. please remove it. Make it configurable using a env variable

Reply:
Updated.